recipes/pygypsy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pygypsy" %}
+{% set version = "0.2.5" %}
+{% set md5 = "7d6d57e422e049974ac4d24da245655e" %}
```

Comment:
Would recommend sha256 over md5. Can be gotten from [pypi.org]( https://pypi.org/ ).